src/Serilog.Sinks.Elasticsearch/Sinks/ElasticSearch/ElasticsearchSinkOptions.cs

```diff
@@ -281,6 +281,11 @@ public int QueueSizeLimit
         /// </summary>
         public RollingInterval BufferFileRollingInterval { get; set; }
 
+        /// <summary>
+        /// Apikey to access elasticsearch, the value is encoded, atob(`id:apikey`).This is another method instead user/password
+        /// </summary>
+        public string Apikey { get; set; }
```

Comment:
```suggestion
        public string ApiKey { get; set; }
```

Lets change the casing to be consistent